release/_publishing.sh

```diff
@@ -35,6 +35,57 @@ function init_gcs {
 	gcloud auth activate-service-account --key-file "${LIFERAY_RELEASE_GCS_TOKEN}"
 }
 
+function update_latest_in_bundles {
```

Comment:
@kiwm, BChan already merged your first pull and also renamed this function to `update_bundles_yml`, please see https://github.com/natocesarrego/liferay-docker/blob/8129817d9c02fa1f453bebc217c5088f24f8cee3/release/_publishing.sh#L219.

Can you please add your logic inside this method?

Thanks